### DISCUSSION TITLE
all not work for me in china

### DISCUSSION THREAD

**Discussion by aohanhongzhi** [shadowsocks/shadowsocks-android#2827] [Q&A]
Please use the forum at https://discourse.shadowsocks.org/c/implementations/shadowsocks-android instead. Your issue will probably be closed.
![Screenshot_20211112-002500_Shadowsocks](https://user-images.githubusercontent.com/37319319/141333809-f09a80da-ca8f-4217-a770-1d8735dbde4b.png)

**Comment by iamsad3508** [comment]
It seems that all your servers are blocked by GFW
